zimui/src/components/channel/tabs/VideosTab.vue

```diff
@@ -3,17 +3,16 @@ import { useMainStore } from '@/stores/main'
 import { computed } from 'vue'
 
 import VideosGridTab from './VideosGridTab.vue'
-import VideosListTab from './VideosListTab.vue'
 
 const main = useMainStore()
-const hideTabs = computed(() => main.channel?.playlistCount === 1)
+const videosAvailable = computed(() => main.channel?.userLongUploadsPlaylist)
 </script>
 
 <template>
-  <div v-if="hideTabs">
-    <videos-grid-tab />
+  <div v-if="videosAvailable">
```

Comment:
Do we need this? Videos tab is anyway not shown when there is not short, isn't it?